centreon/www/include/Administration/configChangelog/viewLogs.php

```diff
@@ -230,6 +244,22 @@ function searchUserName($username)
                 CentreonUtils::ESCAPE_ALL_EXCEPT_LINK
             );
 
+            $author = empty($contactList[$res['log_contact_id']])
+                ? _("unknown")
+                : $contactList[$res['log_contact_id']];
+
+            $element = [
+                'date' => $res['action_log_date'],
+                'type' => $res['object_type'],
+                'object_name' => $objectName,
+                'action_log_id' => $res['action_log_id'],
+                'object_id' => $res['object_id'],
+                'modification_type' => $tabAction[$res['action_type']],
+                'author' => $author,
+                'change' => $tabAction[$res['action_type']],
+                'badge' => $badge[$tabAction[$res['action_type']]],
```

Comment:
Comment : Are you sure that $res have these keys all the time ? no isset here ?

Reply:
cannot say with 100% certainty, but it seemed to be working seemlessly in the previous iteration of this file

Reply:
If you say : "cannot say with 100% certainty", I would recommend you to use isset() in this case. As you wish :)

Reply:
Done